src/SmartComponents/RunTaskModal/ParameterTextbox.js

```diff
@@ -0,0 +1,71 @@
+import React, { useState } from 'react';
+import propTypes from 'prop-types';
+import {
+  Form,
+  FormGroup,
+  FormHelperText,
+  HelperText,
+  HelperTextItem,
+  TextInput,
+} from '@patternfly/react-core';
+
+const ParameterTextbox = ({
+  parameter,
+  updateParameters,
+  customDescription = undefined,
+}) => {
+  const [parameterText, setParameterText] = useState(
+    parameter.default || parameter.value
+  );
+  const [validated, setValidated] = useState('default');
+  const [helperText, setHelperText] = useState(
+    customDescription || parameter.description
+  );
+
+  return (
+    <Form className="pf-v5-u-pb-lg">
```

Comment:
We should avoid using a nested Form here, see my earlier comment

Reply:
Done!